src/lib/structures/PreconditionStore.ts

```diff
@@ -21,10 +21,10 @@ export class PreconditionStore extends Store<Precondition> {
 						message: `The precondition "${precondition.name}" is missing a "messageRun" handler, but it was requested for the "${command.name}" command.`
 				  });
 
-			if (!result.success) return result;
+			if (!result.isErr()) return result;
```

Comment:
Isn't this supposed to be isOk()? Same for the rest

Reply:
or `result.isErr` without the `!`